en/flight_controller/crazyflie2.md

```diff
@@ -94,19 +94,19 @@ For systems that support [make](https://www.gnu.org/software/make/manual/html_no
  * `make venv`
  * Activate the virtual environment: `source venv-cflib/bin/activate`
 
-To launch cfbridge.py everytime:
+To **launch cfbridge** everytime:
 - Switch on CF2 (which is already flashed with PX4 firmware) by pressing its ON button and wait for it to boot up.
 - Connect a Crazyradio PA via USB.
 - Navigate to the crazyflie-lib-python folder.
 - Activate the environment: `source venv-cflib/bin/activate` or `source venv/bin/activate` depending on the name of the venv folder you have.
 - `cd examples`
-- `python cfbridge.py`
+- Launch cfbridge: `python cfbridge.py`
```

Comment:
should we add your script to start the bridge?

Reply:
We can't because depending on which method users follow to build the virtual env, they will have a differently named venv folder.